app/plugins/oauth2-authentication.md

```diff
@@ -99,6 +99,8 @@ Endpoint                     | description
 ---                         | ---
 `/oauth2/authorize`          | The endpoint to the Authorization Server that provisions authorization codes for the [Authorization Code][authorization-code-grant] flow, or the access token when the [Implicit Grant][implicit-grant] flow is enabled. Only `POST` is supported.
 `/oauth2/token`              | The endpoint to the Authorization Server that provision access tokens. This is also the only endpoint to use for the [Client Credentials][client-credentials] and [Resource Owner Password Credentials Grant][password-grant] flows. Only `POST` is supported.
+`/oauth2_tokens`             | The endpoint to the Authorization Server that allows creating new tokens (with `POST`). Useful on migrations (see below).
+`/oauth2_tokens/:token_id`   | The endpoint to the Authorization Server that allows reading (with `GET`), modifying (`PATCH`) and deleting (`DELETE`) access tokens.
```

Comment:
I'd drop the "with" @ `GET`, since Patch and Delete also don't have it.

Reply:
Thanks for the review, I have removed the HTTP methods from the descriptions